cmd/virt-handler/virt-handler.go

```diff
@@ -155,6 +166,19 @@ func (app *virtHandlerApp) Run() {
 	if err != nil {
 		panic(err)
 	}
+
+	app.markNodeAsUnschedulable(logger)
+
+	go func() {
+		sigint := make(chan os.Signal, 1)
+
+		signal.Notify(sigint, syscall.SIGTERM)
+
+		<-sigint
+
+		app.markNodeAsUnschedulable(logger)
```

Comment:
Don't you have to exit here? I would expect that otherwise we would get a sigkill when the grace period is over.

Reply:
Yeah, you are correct, i missed that. I have update it